### PR TITLE
Fix some case issue

### PR DESCRIPTION
**Pull request by chymy**
Signed-off-by: chymy <chang.min1@zte.com.cn>

**What type of PR is this?**
/kind cleanup

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #15863

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

**Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.**:

```docs

```

### CODE REVIEW

**Comment by chymy** [comment]
/assign @jingxu97
Thanks!

**Comment by chymy** [comment]
/assign @gnufied 
Thanks!

**Comment by chymy** [comment]
/assign @ saad-ali
Thanks!

File: pkg/volume/awsebs/aws_util.go [hunk 1]
```diff
@@ -115,7 +115,7 @@ func (util *AWSDiskUtil) CreateVolume(c *awsElasticBlockStoreProvisioner, node *
 	labels, err := cloud.GetVolumeLabels(name)
 	if err != nil {
 		// We don't really want to leak the volume here...
-		klog.Errorf("error building labels for new EBS volume %q: %v", name, err)
+		klog.Errorf("Error building labels for new EBS volume %q: %v", name, err)
```
Comment (oomichi) [inline pkg/volume/awsebs/aws_util.go:118]: Do we have some rules like
- Use uppercase for the first char of `klog.Errorf()` message
- Use lowercase for the first char of `fmt.Errorf()` message

like this PR?

/cc @oomichi

Comment (chymy) [reply pkg/volume/awsebs/aws_util.go:118]: https://github.com/golang/go/wiki/CodeReviewComments#error-strings

Comment (oomichi) [reply pkg/volume/awsebs/aws_util.go:118]: That is only for fmt.Errorf()
Where is for klog.Errorf()?

Comment (chymy) [reply pkg/volume/awsebs/aws_util.go:118]: `klog.Errorf()` is line-oriented and not combined inside other messages, so it should be capitalized (unless beginning with proper nouns or acronyms).

Comment (krmayankk) [reply pkg/volume/awsebs/aws_util.go:118]: @oomichi its standard golang convention https://github.com/golang/go/wiki/CodeReviewComments#error-strings

Comment (krmayankk) [reply pkg/volume/awsebs/aws_util.go:118]: the idea is that fmt.Errorf can return error context that can be used inside other error contexts to build full error context and in that regard the first letter being small , seems good as complete sentences in the overall context strings. for klog.Errorf(), its still just an error level logging so first letter being capital is fine

Comment (oomichi) [reply pkg/volume/awsebs/aws_util.go:118]: Thanks for your explanation @krmayankk @chymy 
At this time, we have a lot of this kid of violation against the standard golang convention under `./pkg` like
```
$ grep "klog.Warningf(\"[a-z]" * -R  | wc -l
114
$ grep "klog.Errorf(\"[a-z]" * -R  | wc -l
250
$ grep "fmt.Errorf(\"[A-Z]" * -R  | wc -l
571
```
Do we want to fix all of them?
If so, it is nice to have some hack script like hack/verify-golint.sh to block such violation anymore.

Comment (oomichi) [reply pkg/volume/awsebs/aws_util.go:118]: statickcheck covers this kind of violation with https://staticcheck.io/docs/checks#ST1005
and this k/k repo contains the check script `hack/verify-staticcheck.sh` but it doesn't detect places which are fixed with this PR..

Comment (chymy) [reply pkg/volume/awsebs/aws_util.go:118]: Not all of them need to be fixed.  We need to check whether any tests require error strings to match or not.

Comment (oomichi) [reply pkg/volume/awsebs/aws_util.go:118]: > We need to check whether any tests require error strings to match or not.

Sorry, I could not follow the above means.
Could you rephrase it?

I confirmed we can enable the above check by removing the following line:
https://github.com/kubernetes/kubernetes/blob/33ee864e9272dcbd043a2782bff6c1281073c810/hack/verify-staticcheck.sh#L39

But if we need to consider what cases should be fixed or not, it would be difficult to enable the check.
I'd like to see what we need to consider here.

Comment (oomichi) [reply pkg/volume/awsebs/aws_util.go:118]: just NOTE: To enable the check, it would be better to specify `-min_confidence 0.6` in kubernetes/hack/verify-golint.sh instead of changing kubernetes/hack/verify-staticcheck.sh

Comment (chymy) [reply pkg/volume/awsebs/aws_util.go:118]: > > We need to check whether any tests require error strings to match or not.
> 
> Sorry, I could not follow the above means.
> Could you rephrase it?

This is part of the error string we found in the test.
